module/Swissbib/src/Swissbib/Controller/DetailPageController.php

```diff
@@ -147,4 +148,60 @@ protected function getResultsManager()
     {
         return $this->serviceLocator->get('VuFind\SearchResultsPluginManager');
     }
```

Comment:
Moved the tag-cloud method to this class, because it is called on an abstract level while it was implemented on child classes only. It also produced code duplication which has been resolved this way

Reply:
Tag Cloud is moved to plugin